handlers/getakt.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Module for requesting documents via AktDirekt, Lantmäteriet
+ * Usage: 
+ * Check for service health, response "UP" or "DOWN" - /origoserver/healthcheck
+ * Get a djvu index page that gets all pages included in akt(document) - /origoserver/document/index.djvu?archive={archive/county-number}&id={akt-number}
+ * Get a single djvu page - /origoserver/document/page_{vers}_{subdoc}_{page}_{archive}_{id}.djvu
+ * 
+ */
+
+// Dependencies
+const url = require('url');
+const http = require('http');
+const httpProxy = require('http-proxy');
+const config = require('../conf/config');
+const getToken = require('../lib/tokenrequest');
+
+// Use the config for AktDirekt
+const akturl = config.getAkt.url;
+const aktkey = config.getAkt.consumer_key;
+const aktsecret = config.getAkt.consumer_secret;
+const aktscope = config.getAkt.scope;
+
+// Token holder
+let token;
+
+// Proxy server holder
+let proxy = httpProxy.createProxyServer();
+
+// Get hold of that nested token and use in proxy request header
+function tokenHeader() { };
```

Comment:
Function definition is missing.

Reply:
I can add a definition to clearify. If there is any better way to get that token for the header I would appreciate the help.